pom.xml

```diff
@@ -298,10 +306,6 @@
                                     <excludes>
                                         <exclude>log4j:log4j</exclude>
                                         <exclude>org.slf4j:slf4j-log4j12</exclude>
-                                        <exclude>org.springframework.boot:spring-boot-starter-logging</exclude>
```

Comment:
why removed?

Reply:
Compilation is banned by Maven enforcer plugin，
Do you have a better suggestion？

   

> maven-enforcer-plugin:3.0.0-M3:enforce (enforce) @ testcontainers-spring-boot ---
   [WARNING] Rule 0: org.apache.maven.plugins.enforcer.BannedDependencies failed with message:
   Found Banned Dependency: org.springframework.boot:spring-boot-starter-logging:jar:2.4.1
   Found Banned Dependency: ch.qos.logback:logback-classic:jar:1.2.3
   Found Banned Dependency: ch.qos.logback:logback-core:jar:1.2.3
   Use 'mvn dependency:tree' to locate the source of the banned dependencies.

Reply:
yes, exclude those modules from dependencies which bring banned ones

Reply:
Do I need to do anything else

Reply:
yes, please leave rules untouched, and remove banned dependencies whoever is bringing them to the project